internal/output/error.go

```diff
@@ -26,15 +25,8 @@ func (e Error) MarshalJSON() ([]byte, error) {
 
 // MarshalHuman implements output.Output
 func (e Error) MarshalHuman() ([]byte, error) {
-	if e.Value == nil {
-		return []byte("\nERROR: Unspecified error"), nil
-	}
-	s := fmt.Sprintf("\nERROR: %v", e.Value.Error())
-	if e.Original != "" {
-		s += fmt.Sprintf(" (%s)", e.Original)
-	}
-
-	return []byte(s), nil
+	// Human readable errors go to stderr. This output package is responsible for stdout output only.
+	return []byte(""), nil
```

Comment:
What if we would return `return nil, errors.New("output package is responsible for stdout output only")` (or some error) here so that we don't expect this function returning something sensible in other parts of code?

Reply:
I think that would require bigger refactor as these `Marshal*` functions are currently called for every commands `Output` without checking the value before call and any error will stop output rendering. This is now special case as we include errors in machine readable normal output (stdout) but omit them from human normal output.